### PR TITLE
Backfill missing lab metadata (30 files)

### PR DESCRIPTION
**Pull request by LuizMacedo**
This PR backfills missing lab metadata in markdown files.

Rules used:
- Add-only (does not overwrite existing metadata keys)
- Keys: lab.title, lab.description, lab.duration, lab.level, lab.islab
- Optional casing normalization to lowercase when enabled

Files changed: 30

- `Instructions/Exercises/AI-01-Explore-LLMs.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/AI-02-RAG.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/AI-03-Multi-stage-reasoning.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/AI-04-Fine-tune-models.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/AI-05-Evaluate-models.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/AI-06-Responsible-AI.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/AI-07-LLMOps.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DE-01-Real-time-ingestion.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DE-02-End-to-end-streaming.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DE-03-Optimize-data-pipelines.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DE-04-Implement-CI-CD-pipeline.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DE-05-Automate-data-ingestion.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DE-06-Implement-data-privacy.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DE-07-Use-SQL-warehouse.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DE-08-Run-notebook-Data-Factory.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DS-03-Train-machine-learning-model.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DS-04-Use-MLflow.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DS-05-Use-AutoML.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DS-06-Tune-hyperparameters.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DS-07-Machine-learning-in-production.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/DS-08-Train-deep-learning-model.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/LA-01-Explore-Azure-Databricks.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/LA-02-Explore-data.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/LA-03-Transform-data.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/LA-04-Explore-Delta-Lake.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/LA-05-Build-data-pipeline.md` (description,duration,level,islab)
- `Instructions/Exercises/LA-06-Build-workflow.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/UC-01-Populate-metastore.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/UC-02-Upgrade-tables-to-unity-catalog.md` (description,duration,level,islab,primarytopics)
- `Instructions/Exercises/UC-03-Secure.md` (description,duration,level,islab)

### CODE REVIEW

**Comment by LuizMacedo** [comment]
Closing this automated metadata PR due to front-matter template compatibility issues. A corrected template-safe run will be submitted.
